source/_includes/asides/hassio_navigation.html

```diff
@@ -10,6 +10,7 @@ <h1 class="title delta">Topics</h1>
           <li>{% active_link /hassio/installing_third_party_addons/ Installing third-party add-ons %}</li>
           <li>{% active_link /hassio/hassos_faq/ Home Assistant OS FAQ %}</li>
           <li>{% active_link /hassio/haos_common_tasks/ Home Assistant OS Common tasks %}</li>
+          <li>{% active_link /hassio/flashing_n2_otg/ Flashing N2+ with OTG-USB %}</li>
```

Comment:
I think that we should instead link this page from the page where we link the ODROID-N2+ as a supported hardware. In the future we will give this guide a better shout out when we redo getting started.